ntbs-service/wwwroot/css/site.scss

```diff
@@ -169,3 +169,16 @@ body {
     outline-offset: 4px;
   }
 }
+
+/* Overriding gratuitous margin on responsive tables */
+.nhsuk-table-responsive {
+    margin-top: 18px;
+    margin-bottom: 40px;
+}
+
+@media (min-width: 40.0625em) {
+    .nhsuk-table-responsive {
+        margin-top: 18px;
+        margin-bottom: 40px;
+    }
+}
```

Comment:
Do we want them shrunk even more on smaller screens?
If not, I **think** we can combine the selectors to something along the lines of:
```
.nhsuk-table-responsive {
  &, @media (min-width: 40.0625em) {
        margin-top: 18px;
        margin-bottom: 40px;
    }
} 
```
... maybe?